photutils/detection/core.py

```diff
@@ -295,39 +313,112 @@ def find_peaks(data, threshold, min_separation=2, exclude_border=True,
         detected peaks exceeds ``npeaks``, the peaks with the highest
         peak intensities will be returned.
 
-    footprint : `~numpy.ndarray` of bools, optional
-        A boolean array where `True` values describe the local footprint
-        region within which to search for peaks at every point in
-        ``data``.  Overrides ``min_separation``, except for border
-        exclusion when ``exclude_border=True``.
+    subpixel : bool, optional
+        If `True`, then a cutout of the specified ``box_size`` or
+        ``footprint`` will be taken centered on each peak and fit with a
+        2D Gaussian (plus a constant).  In this case, the fitted local
+        centroid and peak amplitude will also be returned in the output
+        table.
+
+    error : array_like, optional
+        The 2D array of the 1-sigma errors of the input ``data``.
+        ``error`` is used only to weight the 2D Gaussian fit performed
+        when ``subpixel=True``.
+
+    wcs : `~astropy.wcs.WCS`
+        The WCS transformation to use to convert from pixel coordinates
+        to ICRS world coordinates.  If `None`, then the world
+        coordinates will not be returned in the output
+        `~astropy.table.Table`.
 
     Returns
     -------
     output : `~astropy.table.Table`
         A table containing the x and y pixel location of the peaks and
-        their values.
+        their values.  If ``subpixel=True``, then the table will also
+        contain the local centroid and fitted peak value.
```

Comment:
It actually returns the Gaussian2D amplitude + constant value.  It's value should be close to the peak pixel value.  I'll update the docs here to make that clearer.